dirsrvtests/tests/suites/mapping_tree/acceptance_test.py

```diff
@@ -0,0 +1,65 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import ldap
+import logging
+import pytest
+import os
+from lib389._constants import *
+from lib389.topologies import topology_st as topo
+from lib389.mappingTree import MappingTrees
+
+DEBUGGING = os.getenv("DEBUGGING", default=False)
```

Comment:
I don't think this boiler plate is needed anymore? Isn't this setup in topologies?

Reply:
Sure I can remove that, it's being generated by create_test.py...

Reply:
we should probably clean up create_test.py then too :)

Reply:
Sorry, that was implied by my previous comment :-p

Reply:
Ahh great, lets do that then (probably should be a seperate PR if we need to backport this change ...)